open_widget_framework/open_widget_framework/widget_serializer.py

```diff
@@ -0,0 +1,157 @@
+from django.core.exceptions import ImproperlyConfigured
+from rest_framework.serializers import ModelSerializer, ValidationError
+from rest_framework.validators import UniqueTogetherValidator
+from django.db.models import CharField
+from django.utils.html import format_html
+
+from open_widget_framework.react_fields import ReactCharField, ReactChoiceField
+from open_widget_framework.models import WidgetInstance, WidgetList
+from open_widget_framework.utils import get_widget_class_dict
+
+
+def get_widget_class_configurations():
+    """
+    Get_widget_class_configurations returns a dictionary mapping the names of the widget classes to their configurations
+    """
+    widget_classes = get_widget_class_dict()
+    return {key: WidgetSerializer().get_configuration_form_spec(key) for key in widget_classes.keys()}
```

Comment:
`get_configuration_form_spec` is a class method, so you shouldn't be instantiating it here.